_data/simple-icons.json

```diff
@@ -125,6 +125,11 @@
             "hex": "000000",
             "source": "https://www.apple.com/itunes/marketing-on-music/identity-guidelines.html#apple-music-icon"
         },
+        {
+            "title": "Apple Pay",
+            "hex": "000000",
+            "source": "https://commons.wikimedia.org/wiki/File:Apple_Pay_logo.svg"
```

Comment:
We'd prefer having the `source` being as official as possible - in this case https://developer.apple.com/apple-pay/marketing/